src/day-1-toy/comp.py

```diff
@@ -26,7 +26,6 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [i for i in x if int(i) % 2 == 0]
```

Comment:
This will output the numbers as strings.

Compare to: y = [int(i) for i in x if int(i) % 2==0]